g3w-admin/templates/base.html

```diff
@@ -273,7 +273,15 @@ <h4><i class="icon fa fa-check"></i> Ok!</h4>
     <!-- Bootstrap-datepicker -->
     <script src="{% static "bower_components/bootstrap-datepicker/dist/js/bootstrap-datepicker.min.js" %}"></script>
     <!-- Bootstrap-datepicker-locals -->
-    <script src="{% static "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker.it.min.js" %}"></script>
+    {% for l in SETTINGS.LANGUAGES %}
+    {% if l.0 != 'en' %}
+        {% with "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker."|add:l.0|add:".it.min.j" as datapicker_lang %}
+    <script src="{% static datapicker_lang %}"></script>
+        {% endwith %}
+    {% endif  %}
+    {% endfor %}
+    <script src="{% static "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker.ro.min.js" %}"></script>
+    <script src="{% static "bower_components/bootstrap-datepicker/dist/locales/bootstrap-datepicker.fr.min.js" %}"></script>
```

Comment:
@wlorenzetti Do you think it can be easily integrated with the current testing structure?

- https://playwright.dev/python/docs/intro
- https://playwright.dev/python/docs/library

I feel like I'm missing what I'm doing with all those manual page refresh..

Reply:
@Raruto it would be nice use playwright for testing, but at the moment is better try to finish the PR for v3.6 G3W-SUITE release.

As far as I have read, the natural plugin for Playtight is for pytest, but basically django and in particular G3W-SUITE use the classic unittest, I have never used PyTest.
There's a module for django https://github.com/mxschmitt/python-django-playwright, but it's dated.. so for now we'll put it on the to-do list.

An alternative that I had used in the past is Selenium: https://www.rockandnull.com/django-selenium/.

So maybe we open an issue with a todo on this topic.